packages/components/src/components/buttons/Button/Button.tsx

```diff
@@ -41,8 +36,10 @@ export const ButtonContainer = styled.button<ButtonContainerProps>`
     display: flex;
     align-items: center;
     justify-content: center;
+    flex-direction: ${({ alignIcon }) => alignIcon === 'right' && 'row-reverse'};
+    gap: ${({ hasIcon }) => (hasIcon ? spacings.xs : 0)}px;   
```

Comment:
When there is no icon, there should be only one item and therefore the `gap` would not apply. So, unless I am missing something, this could be just `gap: spacingsPx.xs`

Reply:
It definitely wouldn't, I intended to have `undefined` not `0` when there is no icon to have no extra css 😓 

33fcc67a76